src/category_theory/limits/over.lean

```diff
@@ -113,17 +124,33 @@ def cones_equiv_inverse (B : C) {J : Type v} (F : discrete J ⥤ over B) :
       { dsimp,
         rw ← f.w j,
         refl }
-    end } }
+    end }, }
 
-/-- (Impl) Pull these out to avoid timeouts. -/
+/-- (Impl) A preliminary definition to avoid timeouts. -/
 @[simps]
 def cones_equiv_functor (B : C) {J : Type v} (F : discrete J ⥤ over B) :
   cone (wide_pullback_diagram_of_diagram_over B F) ⥤ cone F :=
 { obj := λ c,
   { X := over.mk (c.π.app none),
     π := { app := λ j, over.hom_mk (c.π.app (some j)) (by apply c.w (wide_pullback_shape.hom.term j)) } },
   map := λ c₁ c₂ f,
-  { hom := over.hom_mk f.hom } }
+  { hom := over.hom_mk f.hom } }.
```

Comment:
Idem

Reply:
The trailing `.` means that Lean doesn't recompile this definition whenever you make edits to the next definition. It saves some time when working on slow proofs. Again, I agree they probably shouldn't make it into the PR.

Reply:
```suggestion
  { hom := over.hom_mk f.hom } }
```

Reply:
Ah this is good to know, that behaviour always confused me!